python/ray/dashboard/modules/metrics/dashboards/train_dashboard_panels.py

```diff
@@ -5,55 +5,63 @@
     Target,
 )
 
+CHECKPOINT_REPORT_TIME_PANEL = Panel(
+    id=1,
+    title="Checkpoint Report Time",
+    description="Time taken to report a checkpoint to storage.",
+    unit="seconds",
+    targets=[
+        Target(
+            expr="sum(ray_train_report_total_blocked_time_s{{{global_filters}}}) by (ray_train_run_name, ray_train_worker_world_rank)",
```

Comment:
these don't seem to take in additional filters like worker id or run id?
Unless that's part of the global filters

Reply:
Currently we just tag the metrics with the `name` and `world_rank` because those are the most understandable by the user (when viewing/filtering metrics). Is that sufficient to map to for the dashboard, or should we also tag with `run_id`? And for the worker would `world_rank` suffice or do we need `actor_id` as well?

Reply:
no i mean filter by. Should these be filtered to a specific run id or worker id when embedded in the dashboard?

Reply:
I think name and world rank will work, if we agree that's what we should pass in.

But name and world rank can be re-used across train runs right? So the metrics embedded on run page wont be scoped only to that specific run anymore. It would be all the runs with that same name

Reply:
Oh actually youre right, we'd technically want to filter exactly by `actor_id`, since there may be multiple run attempts within a run, and multiple runs with the same run name. What's the right way to make it filterable by `worker_id`/`run_id`/`attempt_id`?

Reply:
do we need to scope to a single run attempt? There's no run attempt page with metrics.

But either way, we need to emit enough labels to be able to identify a specific run, run attempt, or worker.

Then grafana dashboard has to add variables for those labels in the train_grafana_dashboard_base.json. These variables should include an All value with default something like `.+`.
Then, the panels need to filter by those variables like run_id=~"$RunId".

Reply:
Gotcha, I think scoping it to single run attempt would be most precise, and if you were to want to see all metrics for rank 0 across attempts you'd just go to the train run page. 

Will try this out!